sdk-setup/src/sdk-chroot

```diff
@@ -294,6 +294,10 @@ prepare_user() {
 
     echo "$user ALL=NOPASSWD: ALL" > ${sdkroot}/etc/sudoers.d/$user
     chmod 0440 ${sdkroot}/etc/sudoers.d/$user
+    
+    if [[ $user == *['!'.]* ]] ; then
+        mv ${sdkroot}/etc/sudoers.d/$user ${sdkroot}/etc/sudoers.d/${user//./}
```

Comment:
How about to simply not use `$user` here, i.e., use a static name for the file, like "sdk"?

Reply:
That's like... too simple :) It would also work with a user named '01_keep_zypp_logfile' :)